configure.ac

```diff
@@ -239,6 +239,15 @@ AM_CONDITIONAL([WITH_PHANTOMJS], [test -n "$PHANTOMJS"])
 AC_PATH_PROG([USERMOD], [usermod], [/usr/sbin/usermod], [$PATH:/usr/local/sbin:/usr/sbin:/sbin])
 AC_DEFINE_UNQUOTED([PATH_USERMOD],["$USERMOD"],[Location of usermod binary])
 
+# newusers
+AC_MSG_CHECKING(whether newusers understands --crypt-method)
```

Comment:
We should look for the binary here and setup a PATH_NEWUSERS as well. Similar to above usermod command.